### PR TITLE
Made 80 letters width + shorted state getter

### PR DESCRIPTION
**Pull request by Piterden**

### CODE REVIEW

File: core/context.js [hunk 1]
```diff
@@ -120,19 +123,16 @@ class TelegrafContext {
   }
 
   get state () {
-    if (!this.contextState) {
-      this.contextState = {}
-    }
-    return this.contextState
+    return (this.contextState = this.contextState || {})
```
Comment (dotcypress) [inline core/context.js]: Not great for code readability

Comment (Piterden) [reply core/context.js]: Why? I think, it is enough clear.
Really not great is https://github.com/telegraf/telegraf/blob/develop/markup.js#L144-L167

Comment (dotcypress) [reply core/context.js]: Because of this: https://eslint.org/docs/rules/no-return-assign

Comment (Piterden) [reply core/context.js]: image
Eslint not shows error, when brackets are present!

Comment (dotcypress) [reply core/context.js]: Yes, but it's just cheating.

Comment (Piterden) [reply core/context.js]: k man. It's your app )

File: core/context.js [hunk 2]
```diff
@@ -120,19 +123,17 @@ class TelegrafContext {
   }
 
   get state () {
-    if (!this.contextState) {
-      this.contextState = {}
-    }
+    this.contextState = this.contextState || {}
```
[the remark this answers is not in the document]
Comment (Piterden) [reply core/context.js]: It's shorter and more readable. Also we rid of one code block.

[the remark this answers is not in the document]
Comment (Piterden) [reply core/context.js]: Take it easy. It's skill from jQuery )))

[the remark this answers is not in the document]
Comment (Piterden) [reply core/context.js]: Check your tg

**Pull request merged**